tests/stylefmt/values/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -16,8 +16,8 @@ exports[`values.css 1`] = `
   border: 1px solid rgba(0, 0, 0, .3);
   font-family: Arial, sans-serif;
   color: rgba(0, 0, 0, 1);
-  margin: 0 20px 0 -24px;
-  background-position: 20% -26px;
+  margin: 0 20px 0 - 24px;
```

Comment:
Before loose mode it would return me

```js
[
  number: '0',
  number: '20px',
  number: '0',
  number: '-24px',
]
```

with loose mode it now gives me

```js
```js
[
  number: '0',
  number: '20px',
  number: '0',
  operator: '-',
  number: '24px',
]
```

But, it gives me `raws: { after: "" }` for the operator so I can workaround.